README.md

```diff
@@ -27,6 +27,54 @@ ts-node examples/calc-ranking.ts fixtures/2017-07-17-lev
 ```
 
 
+Installation
+------------------------------------------------------------------------------
+To install aeroscore you need to install:
+
+Node.js
+```bash
+curl -sL https://deb.nodesource.com/setup_10.x | sudo -E bash -
+sudo apt-get install -y nodejs
```

Comment:
I'd recommend to install Node using https://github.com/creationix/nvm

Reply:
OK.